tests/migrations/test_writer.py

```diff
@@ -454,6 +454,22 @@ def test_sorted_imports(self):
             output
         )
 
+    def test_exclude_models_import(self):
+        """
+        #24514 - Test imports exclude django.db.models if it isn't needed.
+        """
+        migration = type(str("Migration"), (migrations.Migration,), {
+            "operations": [
```

Comment:
I'd leave `'operations'` an empty list. Also, can you use single quotes here, please.